src/core/server/metrics/collectors/types.ts

```diff
@@ -19,6 +20,10 @@ export interface MetricsCollector<T> {
  * @public
  */
 export interface OpsProcessMetrics {
+  /** pid of the kibana process */
+  pid: number;
+  /** name of process (example: 'coordinator' | 'server_worker' | 'task_worker' | 'reporting_worker') */
+  name: 'coordinator' | 'server_worker';
```

Comment:
Since we don't know exactly what our naming scheme will be yet, do you think it's necessary to have a `name` field, vs excluding it and just taking the first/only item from the `processes` array for the time being?

The names here seem like they'd make sense, but if we exclude them we wouldn't have to worry about whether folks are relying on them being stable (since we don't know exactly when multi-process Kibana will be a thing). Could save us from a possible breaking change later.

Reply:
makes sense I'll remove the field so we can introduce it later when needed.